frontend-maven-plugin/src/main/java/com/github/eirslett/maven/plugins/frontend/mojo/InstallNodeAndYarnMojo.java

```diff
@@ -16,6 +18,8 @@
 @Mojo(name = "install-node-and-yarn", defaultPhase = LifecyclePhase.GENERATE_RESOURCES, threadSafe = true)
 public final class InstallNodeAndYarnMojo extends AbstractFrontendMojo {
 
+    private static final String YARNRC_YAML_FILE_NAME = ".yarnrc.yml";
```

Comment:
the README you changed says `yaml`, probably safer to check both file extensions

Reply:
Good catch, my mistake. According to the Yarn documentation (https://yarnpkg.com/configuration/yarnrc) the file has to be named `.yarnrc.yml`, not `.yanrc.yaml`
I'll update the README